### ISSUE TITLE
[BUG] session.total_laps returns the original number of laps.

### ISSUE THREAD

**Issue by Carol52x** [theOehrly/Fast-F1#653]
### Describe the issue:

For the recent race (Brazilian Grand Prix), the number of laps went from 71 to 69 due to an extra formation lap following an aborted start. Shouldn't `session.total_laps` return 69 instead of 71 (which is the original lap count) for this weekend or is there any alternative function that can extract the total number of *racing* laps?

### Reproduce the code example:

```python
`print(session.total_laps)`
```

### Error message:

```shell
71
```

**Comment by theOehrly** [comment]
`Session.total_laps` intentionally returns the original lap count. There are other ways for getting the true count of racing laps, as you have already noticed too.
Also, this is stated explicitly in the documentation ("Originally scheduled number of laps for race-like sessions[...]"). Therefore, I don't see an issue here.

I would assume that the `get_driver().dnf`issue has resolved itself by now? This was likely caused by data being not yet available on ergast.com. Or does this issue still persist?

**Comment by Carol52x** [comment]
Yeah, the issue involving `get_driver().dnf` has resolved. Thank you.
